tests/models/align/test_modeling_align.py

```diff
@@ -65,7 +65,7 @@ class AlignVisionModelTester:
     def __init__(
         self,
         parent,
-        batch_size=13,
+        batch_size=12,
```

Comment:
Same as CLIP, due to the fact that `logits_per_text` and `logits_per_image` being of shape (batch_size, batch_size).